astropy/io/fits/setup_package.py

```diff
@@ -42,6 +42,8 @@ def _get_compression_extension():
                 '-Wno-declaration-after-statement'
             ])
 
+            cfg['define_macros'].append(('HAVE_UNISTD_H', None))
```

Comment:
I thought this setting is supposed to be for `darwin` only, but the placement here seems to imply otherwise. Can you please clarify?

Reply:
The conditional is
[`#if defined(unix) || defined(__unix__)  || defined(__unix) || defined(HAVE_UNISTD_H)`](https://github.com/astropy/astropy/blob/master/cextern/cfitsio/lib/group.c#L27)
so on Linux this should be inconsequential, but not sure about Windows – is the windows 1809 test on Travis running with `ASTROPY_USE_SYSTEM_CFITSIO=0`?

Reply:
Yeah, for Linux this should make no difference, and for Windows there is another include:

https://github.com/astropy/astropy/blob/392d2213475650657d82886b708150355ba01b96/cextern/cfitsio/lib/group.c#L23-L29

Reply:
Ah, I had missed the comment that `direct.h` defines `getcwd` as well.
I was rather concerned if it would try to include a `unistd.h` that does not exist on Windows; seeing now this is actually explicitly disabled:
https://github.com/astropy/astropy/blob/392d2213475650657d82886b708150355ba01b96/astropy/io/fits/setup_package.py#L29-L39

Reply:
Thanks for the clarifications!